pkgs/by-name/re/rep-grep/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rep-grep";
+  version = "unstable-2024-02-06";
```

Comment:
```suggestion
  version = "0-unstable-2024-02-06";
```

Reply:
Dang it, `nix-init`, you misled me.

Reply:
Wait, did it?

```
$ rg -F 'version = "unstable' pkgs/by-name/ -l | wc -l
129

$ rg -F 'version = "0-unstable' pkgs/by-name/ -l | wc -l
32
```

But I see you're following https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming. It's a little idiosyncratic, but I don't care.

Reply:
And of course it invalidates the cargoHash. 😮‍💨